media-libs/libcamera/libcamera-0.6.0.ebuild

```diff
@@ -0,0 +1,120 @@
+# Copyright 2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{13..14} )
+
+inherit meson python-any-r1
+
+DESCRIPTION="A complex camera support library for Linux, Android, and ChromeOS"
+HOMEPAGE="https://libcamera.org"
+SRC_URI="https://gitlab.freedesktop.org/camera/libcamera/-/archive/v${PV}/libcamera-v${PV}.tar.bz2 -> ${P}.tar.bz2"
+S="${WORKDIR}/libcamera-v${PV}"
+
+LICENSE="Apache-2.0 CC0-1.0 BSD-2 CC-BY-4.0 CC-BY-SA-4.0 GPL-2+ GPL-2 LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~arm64"
+IUSE="drm gui gnutls openssl gstreamer jpeg tiff tools qt6 sdl trace +udev elfutils unwind v4l test"
+REQUIRED_USE="qt6? ( tiff gui )"
+
+COMMON_DEPEND="
+	dev-libs/libyaml:=
+	openssl? ( dev-libs/openssl:= )
+	gnutls? ( net-libs/gnutls:= )
+	gstreamer? (
+		>=media-libs/gstreamer-1.14.0:1.0
+		>=media-libs/gst-plugins-base-1.14:1.0
+	)
+	tools? (
+		dev-libs/libevent:=
+		drm? ( x11-libs/libdrm:= )
+		gui? (
```

Comment:
I'm still not certain about the nesting here and would like to hoist it out but it needs to be checked first.